_posts/2015-03-20-one-year-in-and-looking-forward.md

```diff
@@ -143,7 +143,7 @@ source projects.
 
 Onward!
 
-[1]: https://18f.gsa.gov/18f/team/culture/2014/03/19/hello-world-we-are-18f/
```

Comment:
We might consider adding a redirect from this link in the `/2014/03/19/hello-world-we-are-18f/` blog config.